### DISCUSSION TITLE
SLOT CARD COMPONENT

### DISCUSSION THREAD

**Discussion by irazetti** [twilio-labs/paste#2797] [Help: Figma] [answered]
Hi!
Every time that the files have any update, our card component get overlayed with the slot component and then we have to manually remove it and is all over the designs. 
Can you please provide a solution to it?
Thanks!

This is an example of a figma file: https://www.figma.com/file/BpKoEFTlqbcAsn98yxq1s0/Aion--Representative-View
![Screenshot 2022-11-02 at 17 15 35](https://user-images.githubusercontent.com/104209586/199559145-59527644-d033-41d0-9dcd-96d9c8346748.png)

**Comment by loreina** [comment, marked as answer]
Hi @irazetti,

It looks like the Card content is currently being placed on top of the slot, than using the slot. 

I suggest watching [this tutorial](https://www.loom.com/share/611790ab893a4a23b81a43a90697b8f1) how to use our slot components. I've also made a mockup of what it could look like when using slots by creating a custom local component: https://www.figma.com/file/SpeLpBoqN2vZv8KWcSn0J1/Paste-Feedback?node-id=2003%3A399316.

Feel free to come by to [office hours](https://docs.google.com/spreadsheets/d/10vD91zdwr9hjgDOV3HSGnoC0LgvYw-thmn3X_kvW0xk/edit#gid=664218049) if you need any additional assistance.
